digdag-core/src/main/java/io/digdag/core/repository/ProjectControl.java

```diff
@@ -101,7 +103,7 @@ private void updateSchedules(
     {
         ImmutableList.Builder<ScheduleWithScheduler> schedules = ImmutableList.builder();
         for (StoredWorkflowDefinition def : defs) {
-            Optional<Scheduler> sr = srm.tryGetScheduler(revision, def);
+            Optional<Scheduler> sr = srm.tryGetScheduler(revision, def, Optional.of(getUnusedKeysCallback()));
```

Comment:
if we throw UnusedConfigException, this doesn't have to be a callback. A boolean flag is good enough.
Callback is useful if ProjectResource.putProject creates a callback object because ProjectResource.putProject can decide the type of the exception class to throw.
I think boolean flag is good enough.

Reply:
I see, thanks, then I fixed to use boolean for that.
https://github.com/treasure-data/digdag/pull/1381/commits/74d9f4e42d7cbbcf8f163dce4edb3d361bd68a5b